src/nav/MainNavBar.js

```diff
@@ -18,30 +16,25 @@ const styles = StyleSheet.create({
   },
   navBar: {
     flexDirection: 'row',
-    height: NAV_BAR_HEIGHT + STATUS_BAR_HEIGHT,
-    backgroundColor: '#fff',
-    paddingTop: STATUS_BAR_HEIGHT,
-    alignItems: 'center',
+    backgroundColor: BRAND_COLOR,
```

Comment:
We should add back a lot of the old styles for the nav bar. It's now much smaller than standard for iOS and alignment of the heading is no longer centered

In general, I think we should put styling changes in separate PRs from functionality changes